tests/console/cpio.pm

```diff
@@ -0,0 +1,67 @@
+#SUSE"s openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Package: cpio
+# Summary: Create and extract archives with cpio, including all the supported archive formats
+# Maintainer: QE-Core <qe-core@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Define the archive formats that will be used
+    my @formats = ("bin", "odc", "newc", "crc", "tar", "ustar", "hpbin", "hpodc");
+
+    # Download a folder with some files that will be used for the test purpose
+    my $testdatadir = "/usr/share";
+    assert_script_run "cd $testdatadir";
+    assert_script_run "wget --quiet " . data_url('console/cpio/topack.tar.gz');
+    assert_script_run "tar -xzvf $testdatadir/topack.tar.gz";
+
+    assert_script_run "mkdir /tmp/archive";
+
+    # Copy the files into an archive using the different archive formats
+    foreach my $format (@formats) {
```

Comment:
Perhaps this looks a bit better (Just a niptick), same for the others: 

```suggestion
    # Copy the files into an archive using the different archive formats
   assert_script_run "mkdir /tmp/archive";
```

Reply:
Hm, I cannot understand the change you are proposing. Here, I create a main folder that will host the different test folders with the different archive types. When this line is within the for loop, it will fail because this folder is already there and I don't need to create each time a new one :)

Reply:
Crapt :) I just meant: Move the assert_scrip_run right under the comment :)
```suggestion

    # Copy the files into an archive using the different archive formats
    assert_script_run "mkdir /tmp/archive";
    foreach my $format (@formats) {
```

Reply:
oki! done :)